dids/src/main/kotlin/web5/sdk/dids/DidIon.kt

```diff
@@ -724,30 +737,122 @@ public class ResolutionException(msg: String) : RuntimeException(msg)
  */
 public data class KeyAliases(
   public val updateKeyAlias: String?,
-  public val verificationKeyAlias: String?,
+  public val verificationKeyAlias: List<String>,
```

Comment:
```suggestion
  public val verificationKeyAliases: List<String>,
```

Can be multiple?

Reply:
Done.

Yes, it can be multiple. You may add multiple verificationMethods to the DID Document. Each one can be used for different purposes (i.e. for different verification relationships).